ChangeLog

```diff
@@ -2,7 +2,9 @@
 
 
 
-### 3.6.2git <- NOTE: the release version number will be 3.6.3 ###
+### 3.6.2git <- NOTE: the release version number will be 3.7.0 ###
+
+- introduced new server lists: Any Genre 2, Choral/Barbershop (#875)
```

Comment:
This comment is wrong, by the way.  What the change does is https://github.com/jamulussoftware/jamulus/pull/885/files#diff-40ab92941994ec9418dbb76ae46d5aef0e4cb8b034f2be2b31562bb9aca8070f
- rename "Default" to "Any Genre 1"; rename "All Genres" to "Any Genre 2"; rename "Genre Classical/Folk/Choral" to "Genre Classical/Folk"
- add "Any Genre 3" and "Genre Choral/Barbershop"